src/pages/console/uibuilder/collections.mdx

```diff
@@ -9,6 +9,23 @@ Collections are any list of repeating items. You can visually create a collectio
 
 Amplify Studio offers a visual way to bind collections to items in your data model. Collection items can be filtered, sorted, or linked to specific records from the content management view.
 
+### GraphQL and DataStore
+
+Amplify Studio supports data binding to all Amplify GraphQL APIs, **with or without DataStore**. [Learn more about DataStore and Conflict Resolution here](/console/data/data-model/#datastore-and-graphql). 
+
+For APIs without DataStore, there are some minor Collection feature limitations: 
+- Collections cannot be [sorted](#set-sort-conditions)
```

Comment:
Sorted should be capitalized for consistency

Reply:
Fixed